src/errors.ts

```diff
@@ -39,25 +36,76 @@ export class EmptyArrayError extends SdkError {
     }
 }
 
-const isError = (arg: any): arg is Error => arg instanceof Error;
-const isPublicKey = (arg: any): arg is VirgilPublicKey => !(arg instanceof Error);
+export class PrivateKeyAlreadyExistsError extends SdkError {
+    constructor() {
+        super(
+            'You already have a private key. Use EThree.cleanup() to delete it. If you delete last copy of private key, you will not able to decrypt any information encrypted for this private key',
+            'PrivateKeyAlreadyExistsError',
+        );
+    }
+}
+
+export class PrivateKeyNoBackupError extends SdkError {
+    constructor() {
+        super("Backup private key doesn't exist", 'PrivateKeyNoBackupError');
+    }
+}
+
+export class MultipleCardsError extends SdkError {
+    constructor(identity: string) {
+        super(
+            `There are several public keys registered with ${identity}, which is not supported.`,
```

Comment:
What to do in this case? Choose a different identity?

Reply:
For now yes, but I'm think maybe show the latest card?

Reply:
Yea, we decide to throw error in all sdk, cause it is unexpected situation